connectors/src/api/webhooks/webhook_slack.ts

```diff
@@ -145,11 +146,9 @@ const _webhookSlackAPIHandler = async (
       slackTeamId: teamId,
     });
 
-    const slackConfigurations = await SlackConfiguration.findAll({
-      where: {
-        slackTeamId: teamId,
-      },
-    });
+    const slackConfigurations = teamId
+      ? await SlackConfigurationResource.listForTeamId(teamId)
+      : [];
```

Comment:
Why the : [] ? should be part of the implementation if useful since it's everywhere?

Reply:
This ternary condition here was just to return an empty array if `teamId` was falesy. I fixed it by narrowing the type of `teamId`. The prototype of `listForTeamId` was already `Promise<SlackConfigurationResource[]>`, but it was implicit.